src/main/kotlin/io/snyk/plugin/DiffPatcher.kt

```diff
@@ -0,0 +1,94 @@
+package io.snyk.plugin
+
+import io.snyk.plugin.ui.jcef.Change
+import io.snyk.plugin.ui.jcef.DiffPatch
+import io.snyk.plugin.ui.jcef.Hunk
+
+class DiffPatcher {
+
+    fun applyPatch(fileContent: String, diffPatch: DiffPatch): String {
```

Comment:
We should do a follow-up story: move the logic to the server-side by calling a command. Flow would be: 
- applyPatch would call a language server command ("snyk.applyUnifiedDiff")
- this would lead to the language server sending a workspace edit with `workspace/applyEdit`
- the advantage is, that we would then be able to use the existing functionality in the IDEs to apply changes to the document.